src/app/pages/final-confirm/final-confirm.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FinalConfirmComponent } from './final-confirm.component';
+
+describe('FinalConfirmComponent', () => {
+  let component: FinalConfirmComponent;
+  let fixture: ComponentFixture<FinalConfirmComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ FinalConfirmComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(FinalConfirmComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
shouldn't need testing file for final project

Reply:
Angular has a cli command that'll make components for you using "ng generate component" and the testing file comes with it.